WIP-CHANGELOG.md

```diff
@@ -13,9 +13,26 @@
 - Reworked configuration handling to use a common config in addition to the service and workload configs. This is handled in the same way as the sc and wc configs, meaning it is split between a default and an override config. Running `init` will update this configuration structure, update and regenerate any missing configs, as well as merge common options from sc and wc overrides into the common override.
 - Updated fluentd config to adhere better with upsream configuration
 - Fluentd now logs reasons for 400 errors from elasticsearch
+- Migrated from OpenDistro for Elasticsearch to OpenSearch.
+  > **_Breaking Changes:_**
+    This will be a breaking change as some API, specifically related to plugins and security, have been renamed in OpenSearch.
+    The impact will be minimal as the function of the API will stay mostly the same, and the configuration will basically works as is, although renamed.
+    The user experience will change slightly as this will replace Kibana with OpenSearch Dashboards, however the functionality remains the same.
+
+  > **_Compatibility:_**
+    OpenSearch is compatible with existing tools supporting ODFE using a compatibility setting, however this will only last for version 1.x.
+    Newer versions of offical Elasticsearch tools and libraries already contain checks against unofficial Elasticsearch and will therefore not work for either ODFE or OpenSearch.
+    Older versions exists that will still work, and the OpenSearch project is working on providing their own set of tools and libraries.
+
+  > **_Migration:_**
+    This will cause downtime for Elasticsearch and Kibana during the migration, and OpenSearch and OpenSearch Dashboards will replace them.
+    Data will be kept by the migration steps, except security settings, any manually created user or roles must be manually handled.
 
 ### Fixed
 
+- Fluentd will now properly detect changes in container logs.
+- The `init` script will now properly generate secrets for new configuration options.
```

Comment:
No, if new properties were added they where given the standard `somelongsecret` as set in the `config/secrets/*` files.
So this is not just for opensearch, but it is the first new addition since secret generation was integrated into `init`. :slightly_smiling_face:

Reply:
Ah okay then skip my comment here then :slightly_smiling_face: